mechanicalsoup/browser.py

```diff
@@ -139,8 +139,16 @@ def post(self, *args, **kwargs):
         Browser.add_soup(response, self.soup_config)
         return response
 
-    def _request(self, form, url=None, **kwargs):
-        """Extract input data from the form to pass to a Requests session."""
+    @classmethod
+    def _get_request_kwargs(cls, method, url, **kwargs):
```

Comment:
`cls` is unused here, right? Why not use `@staticmethod` instead of `@classmethod` then?

Reply:
Thanks for the good point. I'll use `@staticmethod` to avoid making `cls` useless.